cpp/include/raft/neighbors/detail/ivf_pq_search.cuh

```diff
@@ -738,14 +738,16 @@ inline void search(raft::resources const& handle,
                    float* distances,
                    IvfSampleFilterT sample_filter = IvfSampleFilterT())
 {
+  // raft::print_device_vector("queries", queries, 100, std::cout);
+  RAFT_LOG_INFO("k %u, n_probes %u", k, params.n_probes);
   static_assert(std::is_same_v<T, float> || std::is_same_v<T, uint8_t> || std::is_same_v<T, int8_t>,
                 "Unsupported element type.");
-  common::nvtx::range<common::nvtx::domain::raft> fun_scope(
-    "ivf_pq::search(n_queries = %u, n_probes = %u, k = %u, dim = %zu)",
-    n_queries,
-    params.n_probes,
-    k,
-    index.dim());
+  // common::nvtx::range<common::nvtx::domain::raft> fun_scope(
```

Comment:
Please keep the nvtx marker, it is very useful for perf analysis.

Reply:
Done